src/pages/InsideContents/services/QuoteService.ts

```
@@ -0,0 +1,71 @@
+      Please provide a detailed quote including:
+      1. A FIXED ESTIMATE (using the format 'FIXED_ESTIMATE: $X,XXX')
+      2. Estimated cost range (min-max)
+      3. Required materials with quantities
+      4. Estimated time to complete
+      5. Brief recommendations for this type of project`;
+  }
+
+  // Extract fixed estimate from response - handles both formats
+  extractFixedEstimate(response: string): string {
+    // Try to match the format without brackets first
+    let match = response.match(/FIXED_ESTIMATE:\s*([\$\£\€][\d,\.]+)/i);
```

ESLint

[error] Unnecessary escape character: \$.

(no-useless-escape)